lib/pdk/cli/update.rb

```diff
@@ -10,14 +10,15 @@ module PDK::CLI
     PDK::CLI.template_ref_option(self)
 
     run do |opts, _args, _cmd|
-      require 'pdk/cli/util'
-      require 'pdk/util'
-      require 'pdk/module/update'
+      # Write the context information to the debug log
+      PDK.context.to_debug_log
 
-      PDK::CLI::Util.ensure_in_module!(
-        message:   _('`pdk update` can only be run from inside a valid module directory.'),
-        log_level: :info,
-      )
+      unless PDK.context.is_a?(PDK::Context::Module)
+        raise PDK::CLI::ExitWithError.new(
+          _('`pdk update` can only be run from inside a valid module directory.'),
+          log_level: :error,
```

Comment:
nit: `PDK::CLI::ExitWithError` defaults to the `:error` log level

Reply:
Fixed